Extend/Payment_Gateway.php

```diff
@@ -424,4 +437,38 @@ private function _addItemToBasket(
         $item->setTaxAmount(new Amount($oArticle->getPrice()->getVatValue(), $oCurrency->name));
         $oBasket->add($item);
     }
+
+    /**
+     * Adds transaction entry to db.
+     *
+     * @param \Wirecard\PaymentSdk\Response\Response $oResponse
+     * @param \Wirecard\Oxid\Extend\Order            $oOrder
+     *
+     * @SuppressWarnings(PHPMD.Coverage)
+     */
+    private function _addTransactionToDb(Response $oResponse, Order $oOrder)
```

Comment:
80% of this you use in notify_handler method _onNotificationSuccess

Reply:
will also be updated with the credit card implementation